### PR TITLE
for show vote the user needs a counted assurance

### PR DESCRIPTION
**Pull request by lbarjak**

### CODE REVIEW

File: src/main/java/org/rulez/demokracia/pdengine/VoteRegistry.java [hunk 1]
```diff
@@ -97,33 +97,48 @@ private void checkIfVotingEnabled(final Vote vote) {
...
+	@Override
+	public JSONObject showVote(final VoteAdminInfo adminInfo) {
+		Vote vote = checkAdminInfo(adminInfo);
+		if (!adminInfo.adminKey.equals(vote.adminKey))
+			checkAssurances(vote);
 
 		return vote.toJson(adminInfo.voteId);
 	}
 
+	private void checkAssurances(final Vote vote) throws ReportedException {
```
Comment (magwas) [inline src/main/java/org/rulez/demokracia/pdengine/VoteRegistry.java]: the throws declaration is unnecessary here, please remove.

**Review by magwas — CHANGES_REQUESTED** [review]
sorry for picking nits

**Review by magwas — APPROVED** [review]
nice work
testcasecount=3

**Comment by lbarjak** [comment]
thx Árpi, it wouldn't have gone alone

**Pull request merged**